site/source/user-manual/service-guides/lightning/lightning-integrations.rst

```diff
@@ -86,7 +86,145 @@ Make sure you are already :ref:`running Tor<connecting-tor>` on your system and
 BlueWallet
 ----------
 
-.. note:: Not available at this time - LNDHub is required
+.. note:: Not available at this time.
+
+.. _lnbits-zeus:
+
+LNbits & Zeus
+-------------
+
+.. note:: LND only
+
+.. note:: Requires Tor to be running on your phone - use :ref:`this guide<tor-os>` if necessary.
+
+Using LnBits allows you to create a wallet that makes use of your node with only an alloted amount of sats. This restriction can be very helpful for if you only want to have a small amount for spending on your phone without making your entire lightning balance available. You can even allow other people to have wallets you create for them - think giving your children an allowance. They can start with a set amount (can be zero) and simply spend what you initially make available. They can also earn more for themselves as they will have the freedom to issue their own invoices - all while making use of your node.
```

Comment:
LNbits, not LnBits.

It might be worht explaining the difference between LNbits + Zeus vs just plain Zeus (linked below)